shopping-cart/shopping-cart-scala/shopping-cart/src/main/scala/com/example/shoppingcart/impl/ShoppingCartLoader.scala

```diff
@@ -34,10 +36,14 @@ abstract class ShoppingCartApplication(context: LagomApplicationContext)
   // Register the JSON serializer registry
   override lazy val jsonSerializerRegistry = ShoppingCartSerializerRegistry
 
-  // Register the ShoppingCart persistent entity
-  persistentEntityRegistry.register(wire[ShoppingCartEntity])
-
   lazy val reportRepository = wire[ShoppingCartReportRepository]
-
   readSide.register(wire[ShoppingCartReportProcessor])
+
+  // Initialize ShoppingCart persistent entity.
```

Comment:
Let's not call this a `persistent entity`. It's better to use `persistent entity` to refer only to Lagom classic persistence and adopt `Aggregate` for CQRS/ES behaviors based on Akka Typed Persistence.
```suggestion
  // Initialize the sharding for the ShoppingCart aggregate.
```